src/compas_rhino/__init__.py

```diff
@@ -298,9 +382,164 @@ def _get_rhino_grasshopperplugin_path(version):
     return gh_path
 
 
+# =============================================================================
+# =============================================================================
+# =============================================================================
+# Rhino IronPython
+# =============================================================================
+# =============================================================================
+# =============================================================================
+
+
+# def _get_default_rhino_ironpython_path(version):
+#     version = _check_rhino_version(version)
+
+#     if compas.WINDOWS:
+#         path = _get_default_rhino_ironpython_path_windows(version)
+
+#     elif compas.OSX:
+#         path = _get_default_rhino_ironpython_path_mac(version)
+
+#     else:
+#         raise Exception("Unsupported platform")
+
+#     if not os.path.exists(path):
+#         path = None
+
+#     return path
+
+
+# def _get_default_rhino_ironpython_path_mac(version):
+#     raise NotImplementedError
+
+
+# def _get_default_rhino_ironpython_path_windows(version):
+#     raise NotImplementedError
+
+
+# =============================================================================
+# =============================================================================
+# =============================================================================
+# Rhino CPython
+# =============================================================================
+# =============================================================================
+# =============================================================================
+
+
+def _get_default_rhino_cpython_path(version):
+    version = _check_rhino_version(version)
+
+    if version != "8.0":
+        raise NotImplementedError
+
+    if compas.WINDOWS:
+        path = _get_default_rhino_cpython_path_windows(version)
+
+    elif compas.OSX:
+        path = _get_default_rhino_cpython_path_mac(version)
+
+    else:
+        raise Exception("Unsupported platform")
+
+    if not os.path.exists(path):
+        path = None
+
+    return path
+
+
+def _get_default_rhino_cpython_path_windows(version):
+    raise NotImplementedError
+
+
+def _get_default_rhino_cpython_path_mac(version):
+    if version == "8.0":
+        return "{}/.rhinocode/py39-rh8/python3.9".format(os.path.expanduser("~"))
```

Comment:
Should we also send a warning when people attempt to do symlink install on Rhino 8, say that we recommend pip install?

Reply:
the symlink install is not available for CPython, but it is still the default (and only) procedure for IronPython installations...